vendor/vendor.json

```diff
@@ -273,5 +279,5 @@
 			"revisionTime": "2017-12-17T18:08:21Z"
 		}
 	],
-	"rootPath": "github.com/prometheus/node_exporter"
+	"rootPath": "github.com/kshvakov/node_exporter"
```

Comment:
You need to remove this change.

In order to test locally on a fork, you should symlink the `github.com/prometheus/node_exporter` to your fork in the go src path.